api/envoy/service/tap/v2alpha/common.proto

```diff
@@ -77,11 +79,55 @@ message OutputConfig {
   // sink types are supported this constraint will be relaxed.
   repeated OutputSink sinks = 1 [(validate.rules).repeated = {min_items: 1, max_items: 1}];
 
-  // [#comment:TODO(mattklein123): Output filtering. E.g., certain headers, truncated body, etc.]
+  // For buffered tapping, the maximum amount of received body that will be buffered prior to
```

Comment:
There will be some stream/buffered selection later I assume?

Reply:
Yes, I'm going to start working on streaming once I get the basics in place.I still think that we always want to have buffering capabilities, mainly because it's easier for the end user to work with / look at in certain cases.